### DISCUSSION TITLE
What is the name of this package and how do I uninstall it?

### DISCUSSION THREAD

**Discussion by birds-swim** [SpiralLinux/SpiralLinux-project#252] [Q&A] [answered]
I'm trying to delete unnecessary packages from my system.  Specifically support for languages other than English.

Since GitHub won't allow attachments, here's a pastebin.

What is the name of this package and how do I delete it and its associated packages?

https://pasteboard.co/JM3age1FBCAD.png

**Comment by geckolinux** [comment, marked as answer]
Hi, that's not a package. It's a custom script to install language packages that weighs in at less than 1KB.

**Reply by birds-swim** [reply]
Okay, great!  Thanks for your reply.  I was searching ages with `apt`, but couldn't figure out what it was.  Thank you for the clarification.

**Comment by birds-swim** [comment]
@geckolinux 
So that I'm not opening another thread to say this, just wanted to say "Thank you!" again for making such a rad distro.

I have a hard time trying to figure out how to tweak Calamares (I have no idea what I'm doing) modules on other distros to get the `btrfs` subvolumes and mount options that I want.  But Spiral makes it a breeze!  Normally, I take a distro like Endeavour/Arch, and then immediately get down to performance optimization after installation.  But gosh, Spiral already includes the stuff I want.  I get `btrfs`, Snapper, `zram`/`zswap`, and `tlp`.  Did I miss something else?

Hey, could I message you privately?  I'd like to ask you a few questions about ISO creation.  I'm currently trying to figure out how to package my own changes with your Spiral Linux Builder Edition onto an installable ISO and share it with my family members looking to avoid Windows 11.  Where did you begin this process?  Where can I go/what can I read to get started on the ISO creation path?  I probably will need Calamares on that ISO for an installer, so reading the Calamares docs is probably a given.  Since you've made **two** Linux distros already, I figured you're the authority on the subject by now, lol.  

Do you have the free time to answer a question like that, at the moment?

**Reply by geckolinux** [reply]
Hi there @birds-swim , really glad to hear that you like SpiralLinux.

On the SpiralLinux website toward the bottom of the page in the **Special thanks to:** section you'll find all the most important links that helped me get started with building Debian ISOs. I also describe the process in detail here:
https://github.com/orgs/SpiralLinux/discussions/228#discussioncomment-4809266

Let me know if you have any more questions. Cheers.

**Comment by harlekin777** [comment]
@geckolinux, a very nice distro! I found this place while searching for a language selector tool for Debian. Would it be possible that you publish this script (I guess, it is yours), so other people might also benefit from it? Thank you very much for your kind consideration!

**Reply by geckolinux** [reply]
Hi there, glad you like it!
The script is technically already published, it's a simple BASH script located at `/usr/sbin/language-support.sh` and it just calls native Debian tools for configuring the system languages.

**Comment by harlekin777** [comment]
Thank you very much!



---Original---
From: ***@***.***&gt;
Date: Fri, Jan 31, 2025 21:26 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [SpiralLinux/SpiralLinux-project] What is the name of thispackage and how do I uninstall it? (Discussion #252)




 
Hi there, glad you like it!
 The script is technically already published, it's a simple BASH script located at /usr/sbin/language-support.sh and it just calls native Debian tools for configuring the system languages.
